src/Sidebar_Command.php

```diff
@@ -15,11 +15,7 @@
  */
 class Sidebar_Command extends WP_CLI_Command {
 
-	private $fields = array(
-		'name',
-		'id',
-		'description'
-	);
+	private $fields = [ 'name', 'id', 'description' ];
```

Comment:
Just wondering why this was changed, previous version or a hybrid would be considered more easily readable.
```suggestion
	private $fields = [
		'name',
		'id',
		'description',
	];
```

Reply:
Since there were not many values, I shifted it to single line, reverted back.